index.html

```diff
@@ -2409,6 +2409,45 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Client Hints
+      </h2>
+      <p>
+        Developers often need to know details about how their web app is being
+        rendered in order to make content-negotiation decisions. Though this
+        information is accessible via JavaScript, when sent as part of a
+        {{Request}}, servers are empowered to do the content-negotiation
+        earlier in the process of setting up the web app, before JavaScript
+        execution takes place. To convey this information, this document
+        provides for the following <a data-cite=
+        "client-hints-infrastructure#client-hint-token">client hint token</a>:
```

Comment:
```suggestion
        provides for the `Sec-CH-Display-Mode` [=client hint token=]:
```

If client hint token is not exported, we might need to ask that sped to do that.

Reply:
Client Hint Token is not exported, requesting…

Reply:
Actually, it is exported: https://respec.org/xref/?term=client+hints+token

I will update to reference the spec